### ISSUE TITLE
Inherit AggregatorV3Interface (Part I)

### ISSUE THREAD

**Issue by bh2smith** [cowdao-grants/cow-amm-lp-oracle#10]
Our LPOracle Contract will become [AggregatorV3Interface](https://github.com/smartcontractkit/chainlink/blob/7ec1d5b7abb51e100f7a6a48662e33703a589ecb/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol)

and thus require implementation of the following methods:

```ts
interface AggregatorV3Interface {
  function decimals() external view returns (uint8);

  function description() external view returns (string memory);

  function version() external view returns (uint256);

  function getRoundData(
    uint80 _roundId
  ) external view returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);

  function latestRoundData()
    external
    view
    returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);
}
```

a first iteration can provide some trivial implementations of decimals, description and version with some revert NotImplemented() on the other two more involved functions (to be introduced separately).

**Comment by lumoswiz** [comment]
As we introduce these features, I think we should head in the direction of:
- decimals() should implement the same values as most chainlink USD price feeds (i.e. 8 decimals).
- name() should reflect: <underlying pool name> + "LP Token / USD". Example: "BCoW AMM 50WETH-50GNO LP Token / USD".

**Comment by bh2smith** [comment]
Closed by #17
